weblate/templates/manage-access.html

```diff
@@ -39,13 +39,31 @@
   <td>{% if user.last_login %}{{ user.last_login|naturaltime }}{% else %}{% trans "Not yet signed in" %}{% endif %}</td>
   {% user_permissions user groups %}
   <td>
+
+<a href="#" data-toggle="modal" data-target="#delete_user" class="btn btn-link btn-xs red" title="{% trans "Remove" %}">
+  {% icon 'delete.svg' %}
+</a>
 <form action="{% url "delete-user" project=object.slug %}" method="post" class="inlineform">
 {% csrf_token %}
 <input type="hidden" name="user" value="{{ user.username }}" />
-<button type="submit" class="btn btn-link btn-xs red" title="{% trans "Remove" %}">
-{% icon 'delete.svg' %}
-</button>
+<div class="modal fade" tabindex="-1" role="dialog" id="delete_user">
```

Comment:
You should include user id in the form id, otherwise there are multiple forms with same ID in the page:
```suggestion
<div class="modal fade" tabindex="-1" role="dialog" id="delete_user_{{ user.id }}">
```

Same has to be abode in the modal toggle link.